client/web/src/components/diff/DiffStat.story.tsx

```diff
@@ -1,49 +1,54 @@
-import { number } from '@storybook/addon-knobs'
 import { DecoratorFn, Meta, Story } from '@storybook/react'
 
 import { WebStory } from '../WebStory'
 
 import { DiffStat, DiffStatSquares, DiffStatStack } from './DiffStat'
 
-const getSharedKnobs = () => ({
-    added: number('Added', 10),
-    changed: number('Changed', 4),
-    deleted: number('Deleted', 8),
-})
-
 const decorator: DecoratorFn = story => <div className="p-3 container">{story()}</div>
 
 const config: Meta = {
     title: 'web/diffs/DiffStat',
     decorators: [decorator],
+    argTypes: {
+        added: {
+            type: 'number',
+            defaultValue: 10,
+        },
+        changed: {
+            type: 'number',
+            defaultValue: 4,
+        },
+        deleted: {
+            type: 'number',
+            defaultValue: 8,
+        },
+    },
 }
 
 export default config
 
-export const CollapsedCounts: Story = () => {
-    const stats = getSharedKnobs()
-    return <WebStory>{() => <DiffStat {...stats} />}</WebStory>
-}
+export const CollapsedCounts: Story = args => (
+    <WebStory>{() => <DiffStat added={args.added} changed={args.changed} deleted={args.deleted} />}</WebStory>
+)
```

Comment:
```suggestion
export const CollapsedCounts: Story<React.ComponentProps<typeof DiffStat>> = args => (
    <WebStory>{() => <DiffStat {...args} />}</WebStory>
)
```

To avoid the warnings about an `any` type, you can specify the type for args (which will also ensure nothing accidentally gets typo'd). With that in place, you could also switch back to using a splat for `{...args}`, if you wanted!